src/main/scala/com/wavesplatform/account/Alias.scala

```diff
@@ -29,8 +29,7 @@ object Alias {
   private def validAliasChar(c: Char): Boolean =
     ('0' <= c && c <= '9') || ('a' <= c && c <= 'z') || c == '_' || c == '@' || c == '-' || c == '.'
 
-  private def buildAlias(chainId: Byte, name: String): Either[ValidationError, Alias] = {
-
+  private[wavesplatform] def buildAlias(chainId: Byte, name: String): Either[ValidationError, Alias] = {
```

Comment:
This should also remain private (see above).

Reply:
There is no additional validation in Alias fromBytes

Reply:
Ok, fair enough!